lib/plugins/amplitude/AmplitudeAdapter.ts

```diff
@@ -7,6 +7,7 @@ export const AMPLITUDE_BASE_MESSAGE_SCHEMA = z
     event_type: z.literal<string>('<replace.me>'),
     event_properties: z.object({}),
     user_id: z.string().uuid().or(z.literal('SYSTEM')),
+    groups: z.record(z.string(), z.any()).optional(),
```

Comment:
I am not completely sure if in this case, it is not possible, but I prefer to use `unknown` instead of `any`, although it is a nitpicky comment, I think for Amplitude it does not matter